lib/widgets/infinite_scroll.dart

```diff
@@ -82,7 +82,7 @@ class InfiniteScroll<T> extends HookWidget {
           if (i == data.value.length) {
             // if there are no more, skip
             if (!hasMore.current) {
-              return const SizedBox.shrink();
+              return const SafeArea(child: SizedBox.shrink());
```

Comment:
This definitely should not be `InfiniteScroll`'s responsibility, you do not know where it will be located